docs/addon/walkthrough/README.md

```diff
@@ -520,19 +532,19 @@ status that can be surfaced in various user interfaces.
 	curl https://raw.githubusercontent.com/kubernetes-sigs/application/master/config/crds/app_v1beta1_application.yaml -o config/crd/app_v1beta1_application.yaml
 	```
 
-1. Add an instance of the Application CR in your manifest:
+2. Add an instance of the Application CR in your manifest:
```

Comment:
I updated this section for guestbook-oprator, but current guestbook-operator doesn't include Aplication CR.
Which is better to implement Application CR on guestbook-operator or omit this section?
Sorry for that I don't know details about Application CR...

Reply:
This section should walk through configuring the guestbook operator to manage an Application CR.  The two reconcile options added (`declarative.WithManagedApplication(r.watchLabels), declarative.WithObjectTransform(addon.TransformApplicationFromStatus)`) handle this. These updates look good to me.

The Application CR is meant to bubble up high level information/groupings of a set of resources in the cluster. It's managed by sig-apps.